packages/eslint-plugin-next/lib/rules/no-html-link-for-pages.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
       : [
           path.join(context.getCwd(), 'pages'),
           path.join(context.getCwd(), 'src', 'pages'),
+          path.join(context.getCwd(), 'build', 'pages'),
```

Comment:
```suggestion
```

The only two valid pages directories currently are `pages` and `src/pages`, was `build/pages` meant to be added here?

Reply:
Removed. Included it because I wanted to investigate why a test was failing but forgot to remove 😅